commands/init.ts

```diff
@@ -73,95 +76,125 @@ const personalAccessKeyConfigCreationFlow = async (env, account) => {
-};
+}
 
 const AUTH_TYPE_NAMES = {
   [PERSONAL_ACCESS_KEY_AUTH_METHOD.value]: PERSONAL_ACCESS_KEY_AUTH_METHOD.name,
   [OAUTH_AUTH_METHOD.value]: OAUTH_AUTH_METHOD.name,
 };
 
-exports.command = 'init';
-exports.describe = i18n(`${i18nKey}.describe`, {
+export const command = 'init';
+export const describe = i18n(`${i18nKey}.describe`, {
   configName: DEFAULT_HUBSPOT_CONFIG_YAML_FILE_NAME,
 });
 
-exports.handler = async options => {
+type InitArgs = CommonArgs &
+  ConfigArgs &
+  TestingArgs & {
+    authType?: string;
+    account?: string;
+    'disable-tracking'?: boolean;
+    'use-hidden-config'?: boolean;
+  };
+
+export async function handler(
+  args: ArgumentsCamelCase<InitArgs>
+): Promise<void> {
   const {
     authType: authTypeFlagValue,
     c: configFlagValue,
     providedAccountId,
     disableTracking,
     useHiddenConfig,
-  } = options;
+  } = args;
   const authType =
     (authTypeFlagValue && authTypeFlagValue.toLowerCase()) ||
     PERSONAL_ACCESS_KEY_AUTH_METHOD.value;
 
   const configPath =
     (configFlagValue && path.join(getCwd(), configFlagValue)) ||
     getConfigPath('', useHiddenConfig);
-  setLogLevel(options);
+  setLogLevel(args);
 
   if (!disableTracking) {
     trackCommandUsage('init', {
       authType,
     });
   }
 
-  const env = options.qa ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD;
+  const env = args.qa ? ENVIRONMENTS.QA : ENVIRONMENTS.PROD;
 
-  if (fs.existsSync(configPath)) {
+  if (fs.existsSync(configPath!)) {
     logger.error(
       i18n(`${i18nKey}.errors.configFileExists`, {
-        configPath,
+        configPath: configPath!,
       })
     );
     logger.info(i18n(`${i18nKey}.logs.updateConfig`));
     process.exit(EXIT_CODES.ERROR);
   }
 
   if (!disableTracking) {
-    await trackAuthAction('init', authType, TRACKING_STATUS.STARTED);
+    await trackAuthAction(
+      'init',
+      authType,
+      TRACKING_STATUS.STARTED,
+      providedAccountId!
+    );
   }
 
   const doesOtherConfigFileExist = configFileExists(!useHiddenConfig);
   if (doesOtherConfigFileExist) {
     const path = getConfigPath('', !useHiddenConfig);
-    logger.error(i18n(`${i18nKey}.errors.bothConfigFilesNotAllowed`, { path }));
+    logger.error(
+      i18n(`${i18nKey}.errors.bothConfigFilesNotAllowed`, { path: path! })
+    );
     process.exit(EXIT_CODES.ERROR);
   }
 
-  trackAuthAction('init', authType, TRACKING_STATUS.STARTED);
-  createEmptyConfigFile({ path: configPath }, useHiddenConfig);
+  trackAuthAction(
+    'init',
+    authType,
+    TRACKING_STATUS.STARTED,
+    providedAccountId!
+  );
+  createEmptyConfigFile({ path: configPath! }, useHiddenConfig);
   //Needed to load deprecated config
-  loadConfig(configPath, options);
+  loadConfig(configPath!, args as CLIOptions);
 
   handleExit(deleteEmptyConfigFile);
 
   try {
-    const { accountId, name } = await CONFIG_CREATION_FLOWS[authType](
-      env,
-      providedAccountId
-    );
+    let accountId: number;
```

Comment:
Remember to declare types for these

Reply:
Done in [be71b49](https://github.com/HubSpot/hubspot-cli/pull/1401/commits/be71b495f9f24049001fda5d7c3328c7479fcdc8)